drive/tests/supporting_files/contract/dashpay/dashpay-contract-with-profile-history.json

```diff
@@ -0,0 +1,146 @@
+{
+  "$id": "AcYUCSvAmUwryNsQqkqqD1o3BnFuzepGtR3Mhh2swLk6",
+  "documents": {
+    "profile": {
+      "$documentsKeepHistory": true,
```

Comment:
I really think we should keep this as documentsKeepHistory, because I feel that documentsKeepRevisions would mean that things should be indexed in queries by revision number and not time.

Reply:
I think we should be able to do both. I mean jumping between revisions not just using time but using revisions as well.